medschool/src/parse.rs

```diff
@@ -200,43 +203,53 @@ pub fn parse_docs_into_set<'a>(
 /// DFS helper function to resolve the references of the types. Returns the docs of the fields of
 /// the field we're currently looking at search till its leaf nodes. The leaf here means a primitive
 /// type for which we don't have a [`PartialType`].
+#[tracing::instrument(level = "trace", ret)]
```

Comment:
```suggestion
#[tracing::instrument(level = Level::TRACE, ret)]
```

For some reason we started doing it like this now.